libobs/obs.h

```diff
@@ -449,6 +449,16 @@ EXPORT bool obs_get_audio_info(struct obs_audio_info *oai);
  */
 EXPORT bool obs_get_audio_info2(struct obs_audio_info2 *oai2);
 
+/**
+ * Sets a bool warning that monitoring device coincides with one of DesktopAudio devices.
+ */
+EXPORT void obs_set_prevent_monitoring_duplication(int enable);
```

Comment:
the comment above suggest a bool, but there only is an int and it is not added to the documentation

Reply:
/**
 * Passes the channel index for a Desktop Audio (1 or 2) when its device coincides w/ the monitoring device.
 * Passes 0 if Destop Audio device and monitoring device are not the same.
 * Sets a bool warning core audio that monitoring device coincides with one of DesktopAudio devices.
 */